x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/flyout_top_level_properties.tsx

```diff
@@ -54,7 +54,18 @@ export function FlyoutTopLevelProperties({ transaction }: Props) {
       val: (
         <ServiceLink
           agentName={transaction.agent.name}
-          query={{ ...query, serviceGroup, environment: nextEnvironment }}
+          query={{
+            kuery: query.kuery,
+            latencyAggregationType,
+            offset: query.offset,
+            rangeFrom: query.rangeFrom,
+            rangeTo: query.rangeTo,
+            comparisonEnabled: query.comparisonEnabled,
+            transactionType:
+              'transactionType' in query ? query.transactionType : undefined,
```

Comment:
shouldn't this always be the transaction type of the transaction?

Reply:
yes, good catch, fixed it here: https://github.com/elastic/kibana/pull/136569/commits/3efa93fdbfa7e8921af874217f42c8bdba52a5b0